docker/docker-compose-test.yaml

```diff
@@ -35,15 +38,29 @@ services:
         tail -f /dev/null
     volumes:
       - ${TEST_DATA_DIR}:/main/test_data
-      - ./apt_requirements.txt:/tmp/apt_requirements.txt
-      - ../../element-lab:/main/element-lab
-      - ../../element-animal:/main/element-animal
-      - ../../element-session:/main/element-session
-      - ../../element-interface:/main/element-interface
-      - ../../element-miniscope:/main/element-miniscope
+      # - ./apt_requirements.txt:/tmp/apt_requirements.txt
+      # - ../element-lab:/main/element-lab
+      # - ../element-animal:/main/element-animal
+      # - ../element-session:/main/element-session
+      # - ../element-event:/main/element-event
+      # - ../element-interface:/main/element-interface
+      # - ../element-miniscope:/main/element-miniscope
       - ..:/main/workflow-miniscope
     depends_on:
       db:
-        condition: service_healthy
+        condition: service_started
+
+  adminer:
+    networks:
+      miniscope:
+    image: adminer
+    restart: always
+    ports:
+      - 8082:8080
+
 networks:
-  main:
\ No newline at end of file
+  miniscope:
+    name: workflow-miniscope-network
+
+# volumes:
+#   mysql-miniscope:
```

Comment:
```suggestion
```

Reply:
Sorry - uncommenting this didn't get included in the last commit. Per Joseph's [comment](https://github.com/datajoint/workflow-miniscope/pull/23#discussion_r916985736), I'm in favor of keeping.